### DISCUSSION TITLE
Create pages repository but not published yet?

### DISCUSSION THREAD

**Discussion by VicoErv** [community/community#21541] [Other Feature Feedback, Questions, & Ideas]
<p>I created repository for Github Pages and it already 2 hours ago but still not deployed yet?</p>
<p>this is my repository <a href="https://github.com/VicoErv/VicoErv.github.io" rel="nofollow noopener">https://github.com/VicoErv/VicoErv.github.io</a></p>

**Comment by foopod** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/e/efddba5c0005c42752fc682c06b5beeb1abc1aac.jpeg" data-download-href="/uploads/short-url/ydXafQQkcbLNbDQo1NGXFhrTjnS.jpg?dl=1" title="repo.JPG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/3768616/181065373-d729838e-229c-4ae3-849d-89e362e8516e.jpg" alt="repo.JPG" data-base62-sha1="ydXafQQkcbLNbDQo1NGXFhrTjnS" width="690" height="331" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/e/efddba5c0005c42752fc682c06b5beeb1abc1aac_2_690x331.jpg, https://aws1.discourse-cdn.com/github/optimized/2X/e/efddba5c0005c42752fc682c06b5beeb1abc1aac_2_1035x496.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/e/efddba5c0005c42752fc682c06b5beeb1abc1aac_2_1380x662.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/e/efddba5c0005c42752fc682c06b5beeb1abc1aac_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">repo.JPG</span><span class="informations">1481×712 62.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>What do the settings look like? Also I think the gh-pages branch feature is deprecated. I haven’t seen that around in a while

**Comment by nishanths** [comment]
<p><a class="mention" href="/vicoerv">@vicoerv</a> Could you share a screenshot of your GitHub Pages &gt; Source section?</p>

**Comment by VicoErv** [comment]
<p>thank you for responses, here my screenshot </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/1ecf9018f4b60bfdb004ed7fecbac71e05f0ffd8.png" data-download-href="/uploads/short-url/4oz1A9Co8HrryRdNXmpm729Flbi.png?dl=1" title="Screenshot_403.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/26959883/181065375-d80b0a67-476f-4fe0-9b91-e3733a9931b3.png" alt="Screenshot_403.png" data-base62-sha1="4oz1A9Co8HrryRdNXmpm729Flbi" width="690" height="355" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/1ecf9018f4b60bfdb004ed7fecbac71e05f0ffd8_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_403.png</span><span class="informations">848×437 18.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by winstliu** [comment, marked as answer]
<p><a class="mention" href="/vicoerv">@vicoerv</a> for user GitHub Pages the branch needs to be ‘master’, not ‘gh-pages’.</p>

**Comment by VicoErv** [comment]
<p>thank you, pushed into <code>master</code> branch and now i can access it.</p>
